wiki/Beatmap/Countdown/ja.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - timer
+---
+
+# カウントダウン
+
+*トーナメント用のカウントダウンタイマーは[トーナメント管理コマンド](/wiki/osu!tourney/Tournament_management_commands)を参照してください。*\
+*カウントダウンに関する規則は[ランキング基準のガイドライン](/wiki/Ranking_Criteria#guidelines)を参照してください。*\
+*その他の用法については、[オフセット（曖昧さ回避）](/wiki/Disambiguation/Offset)を参照してください。*
+
+カウントダウンが有効な[ビートマップ](/wiki/Beatmap)は、最初のオブジェクトが表示される前に始まるまでのカウントが表示されます。これは、突然始まる曲に便利ですが、ほとんどのビートマップではこれを有効にする必要はありません。カウントダウンは[song setup](/wiki/Client/Beatmap_editor/Song_Setup)のDesignタブからオン/オフを切り替えることができますが、osu!taikoビートマップでは許可されていません。
+
+カウントダウンアニメーションは`Countdown Speed`(`Half`, `Normal`, `Double`)と`Countdown Offset`で制御できます。`Countdown Speed`は、カウントする速度が左から0.5倍、1倍、2倍になります。`Countdown Offset`は最初の[ヒットオブジェクト](/wiki/Hit_object)の何[拍](/wiki/Beatmapping/Beats_per_minute)前に開始するかを設定します。例えば、オフセット2に設定したカウントダウンでは、カウントダウンの開始が通常より2拍早くなります。イントロ付きのマップの場合、スキップするとカウントダウン前に直接移動します。
+
+カウントダウンアニメーションはスキンで変更することができます。
```

Comment:
missing link

for ref:

`The countdown animation can be [skinned](/wiki/Skinning/Interface#countdown).`